cypress/integration/linkEmbedder.spec.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+describe('links to embeddable content', () => {
+  beforeEach(() => {
+    cy.visitTestEditor()
+  })
+
+  it('GitHub Gist', () => {
+    cy.codemirrorFill('https://gist.github.com/schacon/1')
+    cy.getMarkdownBody()
+      .find('span.one-click-embedding.gist-frame')
```

Comment:
And what if we switch from span to div? That would invalidate this test. The `span` selector is redundant here.

Reply:
Done